internal/engine/postgresql/pg_catalog.go

```diff
@@ -36743,5 +36743,36 @@ func genPGCatalog() *catalog.Schema {
 			},
 		},
 	}
+
+	s.Tables = []*catalog.Table{
```

Comment:
I just overwrote the `Tables` slice to make the output purposefully smaller - just to debug.